### DISCUSSION TITLE
Multilingual docs don't work with mike versioning

### DISCUSSION THREAD

**Discussion by berkantsahn** [squidfunk/mkdocs-material#5330] [Q&A]
I am trying to add versioning to my test project with mike but i couldn't make it. I created multiple yml files for multi language. How can I add versioning to this project?

My test project link: https://github.com/berkantsahn/mkdocs-test

**Comment by kamilkrzyskow** [comment]
I see that this discussion thread got forgotten/missed 👀 I guess you can't delete it. 
So let's link back to other discussions that are a continuation:
(Can't run mike versioning) https://github.com/squidfunk/mkdocs-material/discussions/5356  
(multi-language docs with mike versioning don't work) https://github.com/squidfunk/mkdocs-material/discussions/5364

Tough I suggested both names I see a grammatical issue in the second one 😅 
Consider changing it to  `Multilingual docs don't work with mike versioning` @berkantsahn ✌️

**Comment by kamilkrzyskow** [comment]
@berkantsahn I meant changing the name for https://github.com/squidfunk/mkdocs-material/discussions/5364

**Reply by berkantsahn** [reply]
i did both the same
